met_explore/preprocessing.py

```diff
@@ -59,7 +59,10 @@ def add_chebi_ids(self):
             logger.info("Adding chebi_ids to peak DF")
             self.peak_df["chebi_id"] = None  # Add empty column to fill in.
             self.peak_df["chebi_name"] = None
+            temp_dict = self.chebi_df.set_index('chebi_id').to_dict()
 
+            new_chebi_ids = []
+            new_inchikeys = []
```

Comment:
Instead of lists I think the new_chebi and the new_inchi should be single entities.